charts/templates/NOTES.txt

```diff
@@ -0,0 +1,29 @@
+Thank you for installing {{ .Chart.Name }}.
+
+Your release is named {{ .Release.Name }}.
+
+To check the status of the release, try:
+
+  $ helm status {{ .Release.Name }}
```

Comment:
We might not need to provide helm instructions like status, get pod or service.

Reply:
get pod and get services are added in the next instruction.